nava/functions.py

```diff
@@ -56,6 +56,7 @@ def nava_help():
     print(OVERVIEW)
     print("Repo : https://github.com/openscilab/nava")
     print("Webpage : https://openscilab.com/")
+    print()
```

Comment:
It is not recommended to leave a print statement empty. To ensure best practice, it is suggested to add a `\n` at the end of the previous line.